src/ripple/ledger/ApplyView.h

```diff
@@ -204,6 +204,14 @@ class ApplyView
         STAmount const& preCreditBalance)
     {
     }
+
+    // Called when the owner count changes
+    // This is required to support PaymentSandbox
+    virtual
+    void adjustOwnerCountHook (AccountID const& account,
+        std::uint32_t cur, std::uint32_t next)
+    {};
```

Comment:
I spoke with Vinnie on skype, but for other reviewers:

`adjustOwnerCountHook` is analogous to `creditHook`: it's used by the payment sandbox to track assets acquired during a payment so those newly acquired assets can't be used on that same payment. `adjustOwnerCountHook` should live in the same place as `creditHook`.